FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+//! [ConfigureAndExecuteFindWithDefaultUI]
+async Task ConfigureAndExecuteFindWithDefaultUIAsync(string findTerm)
+{
+    try
+    {
+        // Check if the webView is already initialized and is an instance of CoreWebView2.
+        if (webView.CoreWebView2 == null)
+        {
+            throw new InvalidOperationException("WebView2 is not initialized.");
+        }
+
+        // Initialize the find configuration with specified settings.
+        var findConfiguration = new CoreWebView2FindConfiguration
+        {
+            FindTerm = findTerm,
+            IsCaseSensitive = false,
+            ShouldMatchWord = false,
+            FindDirection = CoreWebView2FindDirection.Forward
+        };
+
+        // By default Find will use the default UI and highlight all matches. If you want different behavior
+        // you can change the SuppressDefaultDialog and ShouldHighlightAllMatches properties here.
+
+        // Start the find operation with the specified configuration.
+        await webView.CoreWebView2.Find.StartFindAsync(findConfiguration);
+
+        // End user interaction is handled via UI.
+    }
+    catch (Exception ex)
+    {
+        // Handle any errors that may occur during the find operation.
+        Console.WriteLine($"An error occurred: {ex.Message}");
+    }
+}
+//! [ConfigureAndExecuteFindWithDefaultUI]
+```
+
+```csharp
+//! [ConfigureAndExecuteFindWithCustomUI]
+async Task ConfigureAndExecuteFindWithCustomUIAsync(string findTerm)
+{
+    try
+    {
+        // Check if the webView is already initialized and is an instance of CoreWebView2.
+        if (webView.CoreWebView2 == null)
+        {
+            throw new InvalidOperationException("WebView2 is not initialized.");
+        }
+
+        // Initialize the find configuration with specified settings.
+        var findConfiguration = new CoreWebView2FindConfiguration
+        {
+            FindTerm = findTerm,
+            IsCaseSensitive = false,
+            ShouldMatchWord = false,
+            FindDirection = CoreWebView2FindDirection.Forward
+        };
+
+        // Specify that a custom UI will be used for the find operation.
+        webView.CoreWebView2.Find.SuppressDefaultDialog = true;
+        webView.CoreWebView2.Find.ShouldHighlightAllMatches = true;
+
+        // Start the find operation with the specified configuration.
+        await webView.CoreWebView2.Find.StartFindAsync(findConfiguration);
+        // It's expected that the custom UI for navigating between matches (next, previous)
+        // and stopping the find operation will be managed by the developer's custom code.
+    }
+    catch (Exception ex)
+    {
+        // Handle any errors that may occur during the find operation.
+        Console.WriteLine($"An error occurred: {ex.Message}");
+    }
+}
+//! [ConfigureAndExecuteFindWithCustomUI]
+```
+
+### Retrieve the Index of the Active Match
+    
+#### Description
+Developers can retrieve the index of the currently active match 
```

Comment:
Will developers need information about the current found word instance other than its ActiveMatchIndex?  For example what longer string it's a part of?

Reply:
Similar to above we don't support this yet but can look at it for the future.